test/tools/yulInterpreter/Interpreter.h

```diff
@@ -186,16 +184,19 @@ class ExpressionEvaluator: public ASTWalker
 	/// stores it in m_value.
 	void evaluateArgs(std::vector<Expression> const& _expr);
 
-	/// Extracts functions from the earlier scopes that are visible for the given function
-	std::map<YulString, FunctionDefinition const*> visibleFunctionsFor(YulString const& _name);
+	/// Finds the function called @a _functionName in the current scope stack and returns
+	/// the function's scope stack (with variables removed) and definition.
```

Comment:
Why are variables removed from the function's scope stack?

Reply:
Because they are not accessible.